app/src/main/java/com/example/cocktailmaster/ui/home/HomeFragment.kt

```diff
@@ -31,7 +32,12 @@ class HomeFragment :
 
     override fun initViews() {
         binding.apply {
-            listOf(imageSearch, imageFavourite, includeCategories.textSeeAllCategory).forEach {
+            listOf(
+                imageSearch,
+                imageFavourite,
+                includeCategories.textSeeAllCategory,
+                includeIngredients.textSeeAllIngredient
```

Comment:
k cần include kia đâu n cx tự hiểu r
```suggestion
                textSeeAllCategory,
                textSeeAllIngredient
```

Reply:
e bỏ include thì nó import cái synthetic như thế này như ltrc a nói
![image](https://user-images.githubusercontent.com/53604984/120738157-8fa8f100-c519-11eb-983d-e21adf306a2c.png)